### DISCUSSION TITLE
Win11 22H2 软件开启效率模式会影响统计吗

### DISCUSSION THREAD

**Discussion by rebron1900** [Planshit/Tai#78] [Q&A]
应该是我多虑了，在后台本来就不统计的，不过还是问问。
![image](https://user-images.githubusercontent.com/7929884/203727753-b471e5b7-32f7-4bcb-b925-03e4e162021b.png)

**Comment by noberumotto** [comment]
理论上有焦点就会统计，但是没测试过诶
